packages/@sanity/form-builder/src/components/Alert.tsx

```diff
@@ -0,0 +1,47 @@
+import {ErrorOutlineIcon, WarningOutlineIcon} from '@sanity/icons'
+import {Box, Card, Flex, Text, ThemeColorToneKey} from '@sanity/ui'
+import React from 'react'
+import styled from 'styled-components'
+
+interface AlertProps {
+  children?: React.ReactNode
+  title: React.ReactNode
+  status?: 'warning' | 'error'
+  suffix?: React.ReactNode
+}
+
+const STATUS_TONES: {[key: string]: ThemeColorToneKey} = {
```

Comment:
This can be improved by using literal types as values (and then the ThemeColorToneKey import can be removed):

```
const STATUS_TONES = {
  warning: 'caution',
  error: 'critical',
} as const
```